package.json

```diff
@@ -61,8 +61,8 @@
     "@material-ui/styles": "4.11.4",
     "classnames": "2.3.1",
     "cozy-bar": "7.16.0",
-    "cozy-client": "27.6.5",
-    "cozy-device-helper": "1.12.0",
+    "cozy-client": "^27.6.5",
+    "cozy-device-helper": "^1.16.1",
```

Comment:
should be squashed with https://github.com/cozy/cozy-notes/pull/260/commits/86b47dec2fdfbef7c4e09efea2ab8c5f00e248e6